### ISSUE TITLE
Libnl detection should be avoided on non Linux systems

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#1360]
**Reported by clopez on 20 May 2014 18:46 UTC**

Libnl is a Linux-only library. Therefore it should be avoided to try to use it on non Linux systems.

I got the following error while trying to build aircrack-ng on Debian/kfreebsd:

```
common.mak:85: *** Cannot find development files for any supported version of libnl. install either libnl1 or libnl3..  Stop.
```

Migrated-From: https://trac.aircrack-ng.org/ticket/1360

**Comment by jbenden** [comment]
**Attachment by clopez on 20 May 2014 18:48 UTC**

Uploaded file [003-dont-abort-libnl-nolinux.diff](https://github.com/aircrack-ng/trac-attachments/blob/master/1360/20140520-184804_171-003-dont-abort-libnl-nolinux.diff)

**Comment by jbenden** [comment]
**Comment by clopez on 20 May 2014 18:46 UTC**

Fixed on aircrack-ng/aircrack-ng@61a3d23b66f6dc7c1f952dfc71b66971845fc8dd
